### ISSUE TITLE
Don't discover any zigbee devices (was working fine previous to reinstalling HA)

### ISSUE THREAD

**Issue by sulliwane** [Koenkk/zigbee2mqtt#23833]
### What happened?

I reinstalled HA on my Armbian box. I setup Z2M, had boot issue about "same ID":

```
2024-09-02T20:18:36.231841000Z [2024-09-02 22:18:36] error: z2m: Error while starting zigbee-herdsman
2024-09-02T20:18:36.239453000Z [2024-09-02 22:18:36] error: z2m: Failed to start zigbee
2024-09-02T20:18:36.246474000Z [2024-09-02 22:18:36] error: z2m: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
2024-09-02T20:18:36.255280000Z [2024-09-02 22:18:36] error: z2m: Exiting...
2024-09-02T20:18:36.269345000Z [2024-09-02 22:18:36] error: z2m: Error: network commissioning timed out - most likely network with the same panId or extendedPanId already exists nearby (Error: AREQ - ZDO - stateChangeInd after 60000ms
2024-09-02T20:18:36.270574000Z at Object.start (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:59:23)
2024-09-02T20:18:36.271437000Z at ZnpAdapterManager.beginCommissioning (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/manager.ts:365:31)
2024-09-02T20:18:36.272531000Z at ZnpAdapterManager.start (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/manager.ts:86:21)
2024-09-02T20:18:36.273259000Z at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:138:29)
2024-09-02T20:18:36.273958000Z at Zigbee.start (/app/lib/zigbee.ts:64:27)
2024-09-02T20:18:36.274628000Z at Controller.start (/app/lib/controller.ts:140:27)
2024-09-02T20:18:36.275407000Z at start (/app/index.js:154:5))

```

So I eventually updated the coordinator FW to [20240710](https://github.com/Koenkk/Z-Stack-firmware/releases/tag/Z-Stack_3.x.0_coordinator_20240710) version ([CC1352P2_CC2652P_launchpad_coordinator_20240710.zip](https://github.com/Koenkk/Z-Stack-firmware/releases/tag/Z-Stack_3.x.0_coordinator_20240710)) using [cc2538-bsl.py](https://github.com/JelmerT/cc2538-bsl). Which didn't fix the issue. 

I ended up pressing the reset button which fixed that issue. Great, Z2M docker container finally starts without complaining.

![image](https://github.com/user-attachments/assets/bd50873d-794c-483e-bed6-29f5d8cdd163)

But when I press the **"Permit Join (all)"** button in the UI, and I put any Zigbee device in pairing mode, nothing happens (**nothing is being discovered or paired by Z2M**) 🤯

I tried changing the channels from 14 to 15 back to 11 to no avail.

Any advice to investigate? It looks like everything is working (my coordinator is correctly recognized and Z2M boots and the UI loads perfectly) but can't discover/pair any device.

Note: all these devices were previously paired with the same coordinator, and worked fine. I just completely reinstalled HA in between, and now it doesn't discover any zigbee device in pairing mode.

Thanks in advance for you help!! 🙏

### Zigbee2MQTT version

1.40.0

### Adapter firmware version

20240701

### Adapter

Sonoff Zigbee Dongle Plus

### Setup

Home Assistant Supervised

**Comment by sulliwane** [comment]
So I've run the following command on my laptop:
```
zigpy radio znp /dev/ttyUSB1 reset
zigpy radio znp /dev/ttyUSB1 form
zigpy radio znp /dev/ttyUSB1 info
zigpy radio znp /dev/ttyUSB1 permit -t 60
```

And it works flawlessly. I can pair a zigbee device.

Plugging the dongle back into my HA box, and restarting zigbee2mqtt addon gives again the error I initially had.

See here for the full DEBUG logs: [z2m-log.txt](https://github.com/user-attachments/files/16857752/z2m-log.txt)

This is my `configuration.yml` file:
```yml
homeassistant: true
advanced:
  homeassistant_legacy_entity_attributes: false
  legacy_api: false
  legacy_availability_payload: false
  log_level: debug
mqtt:
  server: mqtt://core-mosquitto:1883
  user: mqtt
  password: password
serial:
  port: /dev/ttyUSB0
frontend:
  port: 8099
device_options:
  legacy: false
```

And this is the config from the UI:
![image](https://github.com/user-attachments/assets/0d1b41ce-b001-459b-af83-c22d4b7a3980)

It's driving me a bit nuts 🤯 Would really appreciate any idea/pointer. Thanks

**Comment by sulliwane** [comment]
did a another reset of the dongle, changed the config to GENERATE again:
```
advanced:
    # Let Zigbee2MQTT generate a network key on first start
    network_key: GENERATE
    # Let Zigbee2MQTT generate a pan_id on first start
    pan_id: GENERATE
    # Let Zigbee2MQTT generate a ext_pan_id on first start
    ext_pan_id: GENERATE
   channel: 11
```
restart Z2M and finally it discovers new devices. omg.

I suggest the whole logic of Z2M could have its own dedicated page in the doc. Like : 

- When starting from a coordinator that has been reseted before
- need to use config with GENERATE for below 3 fields ([see example](https://github.com/Koenkk/zigbee2mqtt/blob/master/data/configuration.example.yaml)) [...]
- Then Z2M will take care of creating a new network, etc...

- When starting from a coordinator that has already a config inside, 
- Z2M will try to match its internal config
- if there's a mismatch, Z2M will complain about network comissioning.

- if there's no mismatch, etc....

Also I kind of expected that Z2M would just take care of everything. I understand there's no perfect/unique decision tree tho...but error message was bit cryptic for a noob, and I had to learn a lot about how coordinator works, flash firmware, what reset means, what is a network key. Which is interesting, but now that I understand a bit more, I feel there could be some "make it work" button that we press when we just want to clean everything. 

Like when it doesn't work and the user don't bother losing all the paired devices. One button that will reset the device, make sure the config will be generated on the next boot and form a new network. Kind of a last resort button/option. 

Anyway, thanks for all the work that has been done. Just pointing out some small improvement :)

**Comment by sulliwane** [comment]
I think I finally went to the bottom of this issue. I actually had two issues, the first was caused by the old config in the dongle conflicting with a different backup and me not using GENERATE.

The second was actually due to this USB hub that I plugged at the back of my HA box next to the dongle (even though my dongle was plugged in a extension USB cable).

![Screenshot_2024-09-04-13-02-44-93_b5f6883d2c20a96c53babc0b4ac88108](https://github.com/user-attachments/assets/cae033e2-477c-487e-bfc0-4c58b93f711c)

It caused the "MAC channel access failure" error which I reported at the beginning. But I didn't have debug enabled so I didn't see the error initially (maybe errors in the logs should always resurface no matter what log_level is in use no?).

basically the USB hub created interference that totally prevented the dongle to work.

so removing the USB hub solved the issue right away (I could pair devices again, set state, etc).
